moon/autorun/server/sv_powerups_init.moon

```diff
@@ -11,6 +11,8 @@ for powerup in *foundPowerups
     print "Including #{powerup}"
     include powerup
 
+CFCPowerups.spawnLocations = include "powerups/config/maps/" .. game.GetMap! .. ".lua"
```

Comment:
Maybe extract `"powerups/config/maps/" .. game.GetMap! .. ".lua"` to a variable of its own?